.github/workflows/_buildpacks-release.yml

```
@@ -151,8 +147,13 @@ jobs:
       - name: Install Pack CLI
         uses: buildpacks/github-actions/setup-pack@v5.3.1
 
-      - name: Save CNB File
-        run: pack buildpack package ${{ matrix.buildpack_artifact_prefix }}.cnb --config ${{ matrix.buildpack_output_dir }}/package.toml --format file -v
+      - name: Save CNB Files
+        run: |
+          for buildpack in $(echo '${{ needs.compile.outputs.buildpacks }}' | jq -c '.[]'); do
+            artifact_prefix=$(echo "${buildpack}" | jq -r '.buildpack_artifact_prefix')
+            output_dir=$(echo "${buildpack}" | jq -r '.buildpack_output_dir')
+            pack buildpack package ${artifact_prefix}.cnb --config ${output_dir}/package.toml --format file -v
+          done
```

(using bash [here string](https://linux.die.net/abs-guide/x15683.html) saves having to do the echo+pipe, the quoting changes are suggestions from shellcheck)

```suggestion
          for buildpack in $(jq -c '.[]' <<< '${{ needs.compile.outputs.buildpacks }}'); do
            artifact_prefix=$(jq -r '.buildpack_artifact_prefix' <<< "${buildpack}")
            output_dir=$(jq -r '.buildpack_output_dir' <<< "${buildpack}")
            pack buildpack package "${artifact_prefix}.cnb" --config "${output_dir}/package.toml" --format file --verbose
          done
```